README.md

```diff
@@ -27,6 +27,23 @@ centrally managing routing policy and fronting service discovery.
 * [linkerd](linkerd/README.md)
 * [namerd](namerd/README.md)
 
+## Quick start ##
```

Comment:
These quickstart instructions are already in linkerd/README.md and namerd/README.md